ui/pages/create-account/connect-hardware/select-hardware.js

```diff
@@ -231,6 +256,52 @@ export default class SelectHardware extends Component {
     );
   }
 
+  renderQRHardwareWalletSteps() {
+    const steps = [];
+    steps.push({
+      title: this.context.t('QRHardwareWalletSteps1Title'),
+      message: this.context.t('QRHardwareWalletSteps1Description'),
+    });
+
+    steps.push({
+      message: (
+        <p>
+          {this.context.t('QRHardwareWalletSteps2Description')}
+          {this.context.t('keystone')}
+          ,&nbsp;
+          {this.context.t('ngrave')}
+          ,&nbsp;
+          {this.context.t('airgapVault')}
```

Comment:
I think we can localize this whole string:
```
t('QRHardwareWalletSteps2Description', [
t('keystone'), 
]);
```

Doing so will be helpful if punctuation is localized differently